package.json

```diff
@@ -30,18 +30,14 @@
     ],
     "activationEvents": [
         "workspaceContains:**/TMC-Readme.md",
-        "workspaceContains:**/.available_points.json",
-        "workspaceContains:**/.tmc_test_results.json",
-        "workspaceContains:**/.tmcproject.yml",
-        "workspaceContains:**/.tmcproject.json",
-        "workspaceContains:**/.tmc.json",
         "onView:tmcView",
-        "onCommand:tmc.addNewCourse",
+        "onCommand:tmc.cleanExercise",
+        "onCommand:tmc.closeExercise",
         "onCommand:tmc.courseDetails",
         "onCommand:tmc.login",
-        "onCommand:tmc.logout",
         "onCommand:tmc.myCourses",
```

Comment:
No activation for commands that aren't otherwise available when not logged in.

Reply:
?